src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/service/FlowMirrorPointDeleteService.java

```diff
@@ -39,11 +39,12 @@ public class FlowMirrorPointDeleteService extends FlowProcessingService<FlowMirr
         FlowProcessingFsmRegister<FlowMirrorPointDeleteFsm>, FlowProcessingEventListener> {
     private final FlowMirrorPointDeleteFsm.Factory fsmFactory;
 
-    public FlowMirrorPointDeleteService(FlowGenericCarrier carrier, PersistenceManager persistenceManager,
-                                        FlowResourcesManager flowResourcesManager, int speakerCommandRetriesLimit) {
+    public FlowMirrorPointDeleteService(
```

Comment:
But I can't fit it into 2 lines. I need 3 because we have limit for max line length - 120
It will look like this

```
    public FlowMirrorPointDeleteService(FlowGenericCarrier carrier, PersistenceManager persistenceManager,
                                        FlowResourcesManager flowResourcesManager, RuleManager ruleManager, 
                                        int speakerCommandRetriesLimit) {
```